packages/blocks/src/embed-linked-doc-block/styles.ts

```diff
@@ -25,12 +25,8 @@ export const styles = css`
     width: 100%;
     height: 100%;
     z-index: 1;
-    cursor: pointer;
```

Comment:
May be it is a mistake?

Reply:
I refered to the cursor style of embed bookmark and embed sync doc. The cursor for both of them are default.